frontend/packages/client/package.json

```
@@ -4,13 +4,14 @@
   "private": true,
   "dependencies": {
     "@creativebulma/bulma-tooltip": "^1.2.0",
-    "@onflow/fcl": "^0.0.77",
     "@onflow/config": "0.0.2",
+    "@onflow/fcl": "^0.0.77",
     "@onflow/six-transfer-tokens": "0.0.8",
     "bulma": "0.9.3",
     "cookies-js": "^1.2.3",
     "draft-js": "^0.11.7",
-    "draftjs-to-html": "^0.9.1",
+    "draft-js-export-html": "^1.4.1",
+    "immutable": "^4.1.0",
```

```suggestion
    "immutable": "^4.0.0",
```

this will reduce the different versions of immutable included in our bundle see my note below for further details